radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/core/api.yaml

```diff
@@ -2462,6 +2486,33 @@ components:
         transaction_identifier:
           hash: ef71a9d6c63444fce6abd2df8fab2755cfb51f6794e578f60d99337193811842
         duplicate: false
+    OlympiaEndStateRequest:
+      type: object
+      required:
+        - network_identifier
+      properties:
+        network_identifier:
+          "$ref": "#/components/schemas/NetworkIdentifier"
+          description: The name of the network.
+      example:
+        network_identifier:
+          network: mainnet
+    OlympiaEndStateResponse:
+      type: object
+      required:
+        - hash
+        - signature
+        - contents
+      properties:
+        hash:
+          description: The hash of the end state bytes
+          type: string
+        signature:
+          description: The node signature of the end state hash
```

Comment:
Super nit-picky, but could this be `The hex-encoded DER signature of the end state content bytes, signed with the node's key`

Reply:
Wait. I've just realised it's not DER encoded... Will post in other place